CMakeLists.txt

```diff
@@ -126,7 +126,7 @@ set(obs-browser_HEADERS
 
 # only allow browser panels on win32 for now -- other operating systems
 # need more testing
-if(WIN32 AND BROWSER_PANEL_SUPPORT_ENABLED)
+if(BROWSER_PANEL_SUPPORT_ENABLED)
```

Comment:
Might as well change the comment to reflect this isn't just WIN32 anymore.

Reply:
Fixed.